### ISSUE TITLE
Multiple entities not supported

### ISSUE THREAD

**Issue by gumbymn** [davwheat/HeliCam-FiveM#3]
Server sync-resources allows only one entity (heli) to utilize the cam / spotlight. Control of one resource assumes control of all subsequent resources. Ergo only one heli can operate at a time

**Comment by davwheat** [comment]
Did you test this?

**Comment by gumbymn** [comment]
Yes. Multiple times. While I was in one polmav and another user was in a second, while one of us attempted to operate the camera and the spotlight, the other could control the others cam/spotlight

**Comment by davwheat** [comment]
Strange. The camera itself isn't synced, only the spotlight. Even so, that should use player IDs to create an array of spotlights on the server. That array is passed to every client.

Apparently, there's no easy was to sync things via the server because there's no stable value that you can use to distinguish clients.

If you can find a solution to this, I'm all ears.

**Comment by gumbymn** [comment]
I will take a look. I was seeing if there's a way to maybe use a range for hooking the sync but not finding an answer to that.
________________________________
